samples/cellular/location/overlay-nrf700x-wifi-scan-only.conf

```diff
@@ -66,3 +66,4 @@ CONFIG_NET_MGMT_EVENT_STACK_SIZE=1024
 # Heap allocation should be changed when CONFIG_LOCATION_METHOD_WIFI_SCANNING_RESULTS_MAX_CNT
 # and CONFIG_NRF_WIFI_SCAN_MAX_BSS_CNT (which should be the same value) are changed.
 CONFIG_HEAP_MEM_POOL_SIZE=40000
+CONFIG_HEAP_MEM_POOL_IGNORE_MIN=y
```

Comment:
Is this also needed for modem_shell?
Or why it wouldn't be?
I'm trying to understand what the minimum value is that this flag wouldn't be needed but cannot find any documentation on that.

Reply:
nRF70 driver sets it a higher values ~150K, so, if a sample uses less than that then this needs to be enabled else you will see a build warning + it will use driver's default causing memory overflows.

> Is this also needed for modem_shell?
Or why it wouldn't be?

Yes, missed that, will add.